### PR TITLE
Fixes #56 allow custom style for text

### LINKED ISSUE #56: Allow custom style to be applied to action text
Right now, it is not possible to modify the text style 
https://github.com/santomegonzalo/react-native-floating-action/blob/master/component/FloatingActionItem.js#L62

It would be fantastic to be able to supply an optional prop for the style as well as some extra props that would be passed along the text component

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: component/FloatingActionItem.js [hunk 1]
```diff
@@ -54,16 +57,19 @@ class FloatingActionItem extends Component {
             {
               backgroundColor: textBackground,
               elevation: textElevation || elevation
-            }]
-          }
+            },
+            textContainerStyle
```
Comment (msevestre) [inline component/FloatingActionItem.js]: Allows to specify how the text container should look like

File: component/FloatingActionItem.js [hunk 2]
```diff
@@ -54,16 +57,19 @@ class FloatingActionItem extends Component {
...
+          ]}
         >
           <Text
             style={[
               styles.text,
               {
                 color: textColor
-              }
+              },
+              textStyle
```
Comment (msevestre) [inline component/FloatingActionItem.js]: allows to specify how the text should look like

File: component/FloatingActionItem.js [hunk 3]
```diff
@@ -54,16 +57,19 @@ class FloatingActionItem extends Component {
...
             ]}
+            {...textProps}
```
Comment (msevestre) [inline component/FloatingActionItem.js]: useful to pass along some properties to the TextComponent

**Comment by santomegonzalo** [comment]
super nice!

Thanks for your time!

**Pull request merged**